resources/public/api/conf/1.0/application.yaml

```diff
@@ -234,5 +432,17 @@ components:
           authorizationUrl: https://api.service.hmrc.gov.uk/oauth/authorize
           tokenUrl: https://api.service.hmrc.gov.uk/oauth/token
           refreshUrl: https://api.service.hmrc.gov.uk/oauth/refresh
-          scopes:
-            tax-free-childcare-payments: Access all aspects of Tax Free Childcare Payments API
+          scopes: 
+           read: tfc-read
+           write: tfc-write
+  schemas:
+    errorResponse:
+      description: Return when an error occurs
+      type: object
+      properties:
+        errorDescription:
+          description: Description of the error
+          type: string
+          example: Invalid input data
```

Comment:
HMRC error responses actually have 2 properties - `statusCode` and `message`. The YAML here should look more like this:
```yaml
      properties:
        statusCode:
          type: integer
          description: An HTTP response code in the range of 400 to 599
        message:
          type: string
          description: Description of the error
      example:
        statusCode: 400
        message: Provided parameters do not match expected format.